MPCAutofill/cardpicker/models.py

```diff
@@ -265,16 +264,21 @@ def get_medium_thumbnail_url(self) -> Optional[str]:
             self.identifier
         )
 
-    def set_tags(self, tags: list[Tag]) -> None:
-        self.tags = tags
-
-    def get_tags(self) -> list[Tag]:
-        return list(map(Tag, self.tags))
-
     class Meta:
         ordering = ["-priority"]
 
 
+class Tag(models.Model):
+    name = models.CharField(unique=True)
+
+    def __str__(self) -> str:
+        return self.name
+
+    @classmethod
+    def get_tags(cls) -> set[str]:
+        return {x.name for x in Tag.objects.all()}

```

Comment:
With just a name, you can't have alternate names for the tags.  
For example, you're not able to create `EXTENDED = "Extended", "ExtendedArt"`, you can just create `Extented = Extended`.

Reply:
my initial thinking was that it should be fine to only define names for data-driven tags rather than alternate names, but upon reflecting i think you're right. will add an array column for defining additional names / aliases

Reply:
addressed in https://github.com/chilli-axe/mpc-autofill/pull/167/commits/301870336fcd8066527bfcf87bee20810b7be750